test/e2e-tests-latest-eventing.sh

```diff
@@ -72,7 +75,11 @@ function generate_latest_eventing_manifest() {
   for yaml in "${!COMPONENTS[@]}"; do
     local config="${COMPONENTS[${yaml}]}"
     echo "Building Knative Eventing - ${config}"
-    ko resolve -P -f ${config}/ | "${LABEL_YAML_CMD[@]}" > ${yaml}
+    if (( IS_PROW )); then
+        ko resolve -P -f ${config}/ | "${LABEL_YAML_CMD[@]}" > ${yaml}
+    else
+        ko resolve -f ${config}/ | "${LABEL_YAML_CMD[@]}" > ${yaml}
+    fi
```

Comment:
Don't use `-P` when the script is running local. Can't push to Docker when image paths are preserved.

Reply:
Well, I switched to Quay and this problem is gone.